build/rule-generator/get-files-metadata.js

```diff
@@ -60,8 +60,13 @@ const getRuleMatchesFileMeta = (
 		const ruleMatchesJs = {
 			name: `${ruleName}-matches.js`,
 			content: `
-			// TODO: Filter node(s)
-			return node;
+			// TODO: Filter node(s)	
+			
+			function myRuleMatches(node) {
```

Comment:
I didn't mean the function should literally be called `myRuleMatches`. It should be the name of the rule but camelcased. So for a rule called `my-rule`, the matches function should be called `myRuleMatches`.